db/seeds.rb

```diff
@@ -3,11 +3,16 @@
 #
 # Examples:
 #
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-Skill.create([{ name: 'skill1', featured: true },
-              { name: 'skill2', featured: true },
-              { name: 'skill3', featured: false },
-              { name: 'skill4', featured: true },
-              { name: 'skill5', featured: false },
+#   cities = City.create([{ name: "Chicago" }, { name: "Copenhagen" }])
+#   Mayor.create(name: "Emanuel", city: cities.first)
```

Comment:
How would you feel about using `factory_girl_rails` to create seeds? :spaghetti: